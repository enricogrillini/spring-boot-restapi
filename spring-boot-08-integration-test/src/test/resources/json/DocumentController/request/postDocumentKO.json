{
  "id" : 1,
  "name" : "doc-1",
  "description" : "Contratto vita - prova inserimento",
  "data" : "2022-01-01",
  "updateBy" : "Pippo"
}