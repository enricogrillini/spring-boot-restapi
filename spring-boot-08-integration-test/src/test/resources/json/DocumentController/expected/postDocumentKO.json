{
  "code" : "BUSINESS_ERROR",
  "description" : "Errore generico",
  "detail" : "Id documento deve essere null in inserimento"
}